controllers/apply/under10k/fields/project-postcode.js

```diff
@@ -8,9 +8,6 @@ const Field = require('../../lib/field-types/field');
 module.exports = function (locale) {
     const localise = get(locale);
 
-    /**
-     * @TODO: Is it worth this being a dedicated PostcodeField type?
```

Comment:
Not worth adding this abstraction at this point, maybe if we build another form with a single post-code field.